RetailSuiteAccounting/UserInterface/01_Book.md

```diff
@@ -0,0 +1,1483 @@
+by default# Post
```

Comment:
vgl. mit Datei
eventuell Auslagern der Funktionen in separate Datei 

[01_Book_hg.md](https://github.com/Actindo-AG/Documentation/files/9596690/01_Book_hg.md)